#!/usr/bin/env python
PACKAGE = "cat_backend"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

#gen.add("int_param", int_t, 0, "An Integer parameter", 50, 0, 100)
#gen.add("double_param", double_t, 0, "A double parameter", .5, 0, 1)
#gen.add("str_param", str_t, 0, "A string parameter", "Hello World")
#gen.add("bool_param", bool_t, 0, "A Boolean parameter", True)

gen.add("planning_group", str_t, 0, "The group to plan for", "right_arm")
gen.add("query_start_state", bool_t, 0, "Show start state", True)
gen.add("query_goal_state", bool_t, 0, "Show goal state", True)
gen.add("marker_scale", double_t, 0, "Scale for interactive markers", .5, 0, 1)

# query_start_state_property_
# query_goal_state_property_

exit(gen.generate(PACKAGE, PACKAGE, "Backend"))


